### DISCUSSION TITLE
Assessment Workflow on SARAL App

### DISCUSSION THREAD

**Discussion by ShreyashiSoni** [Sunbird-Saral/Community#75] [Polls]
Hello everyone, I have prepared current workflow for the Assessment use case in the SARAL App. Please review it and let me know if it appears satisfactory. Furthermore, I would like your input regarding the last step in the workflow (marked in yellow). Please vote on whether we should retain this functionality or if it is not required.

![Issue Tree_SARAL@1 125x (6)](https://github.com/Sunbird-Saral/Community/assets/129832399/bfc61472-dfb7-4a39-9f52-7c90eb60a365)

This should give clarity to all of us on assessment workflow we are following. Thanks.

@surendrasinghs

**Comment by ShreyashiSoni** [comment]
Hi Suren, As per our discussion yesterday I have added the mock ups for functionality of viewing list of students whose OMRs have not yet scanned. Please go through it and let me know of your thoughts. 

![iPhone 14 Pro - 35](https://github.com/Sunbird-Saral/Community/assets/129832399/1e9f275c-fd0f-4372-8432-4f73e175ccca)
![iPhone 14 Pro - 36](https://github.com/Sunbird-Saral/Community/assets/129832399/d97f7983-cd02-407a-8507-bf364aa4f24c)


@surendrasinghs

**Reply by ShreyashiSoni** [reply]
@surendrasinghs Hi Suren, please let me know your thoughts on the above mocks ups.

**Reply by kashyapnitin** [reply]
So my suggestions would be to make this whole screen a bit more intuitive
Have the table contain the following rows in this order
Total Students : 55
Students marked Present: 52

Option A
Scans completed & submitted: 20 (starts of as 0)
Scans completed, yet to be submitted : 20 (starts of as 0)
Scans yet to be completed: 12 (starts of as 52)

or Option B
Student data scanned & submitted: 20 (starts of as 0)
Student data scanned, yet to be submitted : 20 (starts of as 0)
Student data yet to be scanned: 12 (starts of as 52)


So that the terminology is very clear and consistent the data for a studnet moves from 1) yet to be scanned/completed to 2) scaned/completed; yet to be submitted 3) scanned/completed & submitted

and the numbers all add up to Students marked present at all time.

Similarly for placing the buttons there are 2 options

Option 1) The right buttons placed next to the right number / row
so Scans completed & submitted only has a View button next to it
Scans completed; yet to be submitted has 2 buttons next to it a) Review/Rescan & b) Submit
Scans yet to be completed has no button next to it as the big scan button is underneath it

Option 2) All the buttons are separate from the summary numbers. In which case I would suggest we 
have the big scan button as. the main CTA
followed by 3 buttons arranged in a single row which read
Submit Completed scans
Review Completed scans
View Submitted scans
 
Would be great if the 2/3 options here can be mocked up and we can get a quick read from some real users w.r.t intuitiveness of the options.

I would personally prefer Option A+ Option 1 for brevity and clarity and keeping the CTAs next to the main information.

**Reply by ShreyashiSoni** [reply]
Thanks Nitin, let me share mockups with 2-3 options as guided by your suggestions. We can decide further on it.
